diff -c -r blockparser/cb/transactions.cpp blockparser_fairbrix/cb/transactions.cpp
*** blockparser/cb/transactions.cpp	2013-03-04 13:29:10.000000000 +0100
--- blockparser_fairbrix/cb/transactions.cpp	2013-04-13 18:30:25.338418396 +0200
***************
*** 78,83 ****
--- 78,86 ----
              #if defined(LITECOIN)
                  const char *addr = "LKvTVnkK2rAkJXfgPdkaDRgvEGvazxWS9o";
                  warning("no addresses specified, using popular address %s", addr);
+             #elif defined(FAIRBRIX)
+                 const char *addr = "fca1JykZfJ85MiX8JfGuyKYQ3pkpjsafBg";
+                 warning("no addresses specified, using example address %s", addr);
              #else
                  const char *addr = "1dice8EMZmqKvrGE4Qc9bUFf9PX3xaYDp";
                  warning("no addresses specified, using satoshi's dice address %s", addr);
diff -c -r blockparser/Makefile blockparser_fairbrix/Makefile
*** blockparser/Makefile	2013-03-04 13:29:10.000000000 +0100
--- blockparser_fairbrix/Makefile	2013-04-13 18:28:41.639722065 +0200
***************
*** 7,12 ****
--- 7,13 ----
  INC =                           \
          -I.                     \
          -DNDEBUG                \
+         -DFAIRBRIX              \
  #        -DLITECOIN              \
  
  COPT =                          \
diff -c -r blockparser/parser.cpp blockparser_fairbrix/parser.cpp
*** blockparser/parser.cpp	2013-03-04 13:29:10.000000000 +0100
--- blockparser_fairbrix/parser.cpp	2013-04-13 18:26:59.617004958 +0200
***************
*** 365,370 ****
--- 365,372 ----
      std::string coinName(
          #if defined LITECOIN
              "/.litecoin/"
+         #elif defined FAIRBRIX
+                 "/.fairbrix/"
          #else
              "/.bitcoin/"
          #endif
***************
*** 512,517 ****
--- 514,521 ----
      static const uint32_t expected =
      #if defined(LITECOIN)
          0xdbb6c0fb
+     #elif defined(FAIRBRIX)
+         0xdbf9dbf9
      #else
          0xd9b4bef9
      #endif
diff -c -r blockparser/util.cpp blockparser_fairbrix/util.cpp
*** blockparser/util.cpp	2013-03-04 13:29:10.000000000 +0100
--- blockparser_fairbrix/util.cpp	2013-04-13 18:25:02.158481456 +0200
***************
*** 430,435 ****
--- 430,437 ----
          memcpy(1+data, hash160, kRIPEMD160ByteSize);
          #if defined(LITECOIN)
              data[0] = 48;
+         #elif defined(FAIRBRIX)
+             data[0] = 95;
          #else
              data[0] = 0;
          #endif
diff -c -r blockparser/util.h blockparser_fairbrix/util.h
*** blockparser/util.h	2013-03-04 13:29:10.000000000 +0100
--- blockparser_fairbrix/util.h	2013-04-13 18:23:28.047664431 +0200
***************
*** 257,262 ****
--- 257,264 ----
          const uint8_t *hash160,
          #if defined(LITECOIN)
                uint8_t type = 48
+         #elif defined(FAIRBRIX)
+               uint8_t type = 95
          #else
                uint8_t type = 0
          #endif
